developer-docs-site/docs/concepts/base-gas.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: "How Base Gas Works"
+title: "Computing Transaction Gas"
 id: "base-gas"
 ---
 
-# How Base Gas Works
+# Computing Transaction Gas
```

Comment:
May be for later - we should point to the doc for the gas profiler here. cc @vgao1996